src/tree/SubscriptionTreeItem.ts

```diff
@@ -88,7 +88,8 @@ export class SubscriptionTreeItem extends SubscriptionTreeItemBase {
 
         LocationListStep.setLocationSubset(wizardContext, Promise.resolve(locations), webProvider);
         LocationListStep.addStep(wizardContext, promptSteps, {
-            placeHolder: localize('selectLocation', 'Select a region for Azure Functions API and staging environments')
+            placeHolder: localize('selectLocation', 'Select a region for Azure Functions API and staging environments'),
+            noPicksMessage: localize('noRegions', 'No available regions.')
```

Comment:
Do you think we could just have the `noPicksMessage` be the default behavior for `LocationListStep`?  It probably doesn't happen to frequently, but don't really see why we should only have this be limited to Static Web Apps.